### PR TITLE
mgr/dashboard: Add linter for unclosed HTML tags

### PR DESCRIPTION
**Pull request by p-na**
Adds `html-hint` to the `npm run lint` command and hence, it's automatically added to `run-frontend-unit-test.sh`, which is called on pull request creation. Ensures closed HTML tags. It found the [issue spotted by @s0nea](https://github.com/ceph/ceph/pull/28899).

```
user@home ~/src/ceph-4/src/pybind/mgr/dashboard/frontend (wip-pna-fix-tag*) $ npx htmlhint -r tag-pair src/app

   /home/user/src/ceph-4/src/pybind/mgr/dashboard/frontend/src/app/app.component.html
      L9 |
          ^ Tag must be paired, missing: [ </block-ui></block-ui> ], open tag match failed [ <block-ui> ] on line 8. (tag-pair)

Scanned 121 files, found 1 errors in 1 files (146 ms)
user@home ~/src/ceph-4/src/pybind/mgr/dashboard/frontend (wip-pna-fix-tag*) $ ..
```

Fixes: http://tracker.ceph.com/issues/40686

Signed-off-by: Patrick Nawracay <pnawracay@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/package.json [hunk 1]
```diff
@@ -99,6 +100,7 @@
     "babel-preset-env": "1.7.0",
     "codelyzer": "4.5.0",
     "html-linter": "1.1.1",
+    "htmlhint": "^0.11.0",
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/package.json]:
```suggestion
    "htmlhint": "0.11.0",
```

Comment (alfonsomthd) [reply src/pybind/mgr/dashboard/frontend/package.json]: +1

**Comment by ricardoasmarques** [comment]
jenkins retest this please

**Comment by ricardoasmarques** [comment]
jenkins test make check

**Comment by ricardoasmarques** [comment]
make check is failing due to a missing "</div>": 

```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.html
[37m      L32 |[90m[39m
[37m           ^ [31mTag must be paired, missing: [ </div> ], open tag match failed [ <div class="row"> ] on line 1. (tag-pair)[39m

Scanned 125 files, found 1 errors in 1 files (367 ms)
```

**Pull request merged**